src/Generator/CreditMemoPdfFileGenerator.php

```diff
@@ -40,18 +36,16 @@ final class CreditMemoPdfFileGenerator implements CreditMemoPdfFileGeneratorInte
 
     public function __construct(
         RepositoryInterface $creditMemoRepository,
-        Environment $twig,
-        GeneratorInterface $pdfGenerator,
+        TwigToPdfGeneratorInterface $twigToPdfGenerator,
```

Comment:
This change is a BC break and we cannot do that this way, you should deprecate the arguments you are removing and a new one as a last and nullable argument. And use the new one only if it is not null. However, I'm not sure if this code is needed according to my other comment.

Reply:
Thanks, I removed BC and did deprecations in commit https://github.com/Sylius/RefundPlugin/pull/361/commits/22f818c861d1c10ba8481875aa8bd560d33814db. However, I worry I overdid it with that many `@trigger_error` ;d